pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -68,6 +68,37 @@ const (
 )
 
 const (
+	activateRawOsdOnNodeScript = `
```

Comment:
How long does this script take to run? Is it worth optimizing this to only run when the device changed?

Reply:
It takes 250-300ms to list a single disk via `ceph-volume raw list <disk>` and `350-450ms` to list all 3 OSDs via `ceph-volume raw list`.

This script runs in less than a second on my 3-disk test node.

Reply:
RE: @leseb 

>Why not list against $ROOK_BLOCK_PATH first and if there is a mismatch, go through an entire listing? This will save processing time.

It will save 100-150ms. I'm not sure that is worth the optimization.